src/main/kotlin/app/revanced/patches/youtube/misc/hdrbrightness/patch/HDRBrightnessPatch.kt

```diff
@@ -40,7 +41,10 @@ class HDRBrightnessPatch : BytecodePatch(
             )
         )
 
-        val method = HDRBrightnessFingerprint.result!!.mutableMethod
+        var result = HDRBrightnessFingerprint.result
+        if (result == null) result = HDRBrightnessLegacyFingerprint.result
```

Comment:
Is it possible to only rely on a single fingerprint which works for both versions?

Reply:
Unfortunately no. On versions prior to 17.33.42 the structure of the method changes completely.